### PR TITLE
Removed Magic Numbers in propagator #5685

### PR DESCRIPTION
**Pull request by kaustubh-welankar**

### CODE REVIEW

**Comment by kaustubh-welankar** [comment]
@mrow4a @jturcotte @guruz @ogoffart

**Comment by jturcotte** [comment]
@mrow4a Maybe it would help to be a bit more specific, because I also don't completely understand what you mean, or how this relates to the title and description in this issue.

**Comment by kaustubh-welankar** [comment]
@mrow4a Removed the changes made to the config files

File: src/libsync/owncloudpropagator.cpp [hunk 1]
```diff
@@ -392,6 +392,12 @@ PropagateItemJob* OwncloudPropagator::createJob(const SyncFileItemPtr &item) {
     return 0;
 }
 
+quint64 OwncloudPropagator::smallFileSize()
+{
+    static quint64 smallFileSize = 100*1024;
```
Comment (ogoffart) [inline src/libsync/owncloudpropagator.cpp]: could be const.    (and does not even need to be static)

Comment (kaustubh-welankar) [inline src/libsync/owncloudpropagator.cpp]: I'll change that as well.

**Comment by kaustubh-welankar** [comment]
@mrow4a Could you accept the pull request or suggest additional changes?

**Comment by guruz** [comment]
👍

**Pull request merged**